openedx/core/djangoapps/content_libraries/serializers.py

```diff
@@ -84,6 +84,32 @@ class LibraryXBlockTypeSerializer(serializers.Serializer):
     display_name = serializers.CharField()
 
 
+class LibraryBundleLinkSerializer(serializers.Serializer):
+    """
+    Serializer for a link from a content library blockstore bundle to another
+    blockstore bundle.
+    """
+    id = serializers.SlugField()  # Link name
+    bundle_uuid = serializers.UUIDField(format='hex_verbose', read_only=True)
+    # What version of this bundle we are currently linking to.
+    # This is never NULL but can optionally be set to null when creating a new link, which means "use latest version."
+    version = serializers.IntegerField(allow_null=True)
```

Comment:
How important do you think it is that we have this capability of specifying `null` to get the latest version? Is that just to match conventions with Blockstore (I honestly don't even remember what we did there at this point)?

Reply:
It's not a blockstore convention, but having some way to say "use latest version" is important for performance I think, because otherwise it's necessary to make additional API calls to determine the latest version, and "use latest version" will be a very common use case.